manager/settings.py

```diff
@@ -113,7 +113,7 @@
 
 USE_L10N = True
 
-USE_TZ = False
+USE_TZ = True
```

Comment:
If you want to use time zones, then you should use Django's own `timezone` module instead of the standard `datetime`.

`from django.utils import timezone` (it has got all the same members as the standard `datetime`).

Reply:
ok